searx/engines/duckduckgo_lite.py

```diff
@@ -0,0 +1,159 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""DuckDuckGo Lite
+"""
+
+from json import loads
+
+from lxml.html import fromstring
+
+from searx.utils import (
+    extract_text,
+    match_language,
+    eval_xpath,
+    eval_xpath_getindex,
+    )
+
+# about
+about = {
+    "website": 'https://lite.duckduckgo.com/lite',
+    "wikidata_id": 'Q12805',
+    "official_api_documentation": 'https://duckduckgo.com/api',
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'HTML',
+}
+
+# engine dependent config
+categories = ['general']
+paging = True
+supported_languages_url = 'https://duckduckgo.com/util/u172.js'
```

Comment:
Perhaps another PR: the last URL is https://duckduckgo.com/util/u588.js 

To get this URL:
* download `https://duckduckgo.com/`
* search for `<script type="text/javascript" src="/util/u([0-9]+).js"></script>`

Reply:
see 3298fbc